environments/dev-env/CONTRIBUTING.md

```diff
@@ -0,0 +1,510 @@
+# KCP-Edge Contributor Development Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
```

Comment:
I would explicitly mention the `git clone` of this repo before any usage of the local copy of this repo.

Reply:
I agree

Reply:
fixed